deployments/frontend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12
```

Comment:
This will result in a huge front-end image, and we don't really need node_moudle in the image, we just need to copy the build output.

Reply:
What is required to run the node server?